tests/cnf/ran/ztp/tests/ztp_argocd_acm_crs.go

```diff
@@ -0,0 +1,155 @@
+package tests
+
+import (
+	"slices"
+	"time"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/argocd"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranhelper"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranparam"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/ztp/internal/helper"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/ztp/internal/tsparams"
+	corev1 "k8s.io/api/core/v1"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+)
+
+var _ = Describe("ZTP Argo CD ACM CR Tests", Label(tsparams.LabelArgoCdAcmCrsTestCases), func() {
+	var (
+		acmPolicyGeneratorImage        string
+		oldAcmPolicyGeneratorContainer corev1.Container
+	)
+
+	BeforeEach(func() {
+		By("checking that the required clusters are present")
+		if !ranhelper.AreClustersPresent([]*clients.Settings{HubAPIClient, Spoke1APIClient}) {
+			Skip("not all of the required clusters are present")
+		}
+
+		By("determining the container image for ACM CR integration")
```

Comment:
Since the new instructions only start with 4.14, would we still need this step in the test to handle versions 4.12 and 4.13? I agree it'll be important to make sure we're dumping the argo CD CRs, they're very helpful for debugging